### DISCUSSION TITLE
To use Blender's text editor for writing huge prompts

### DISCUSSION THREAD

**Discussion by rpserge** [carson-katri/dream-textures#353] [Ideas]
![image](https://user-images.githubusercontent.com/45179039/198118364-d6dc3866-0181-4176-85ed-1001ecba9e11.png)

**Comment by carson-katri** [comment]
This was implemented in #332

**Reply by rpserge** [reply]
Oh wow! Haven't seen it yet! Amazing! Thank you so much!
